STUDIO/org.openl.rules.webstudio/webapp/pages/modules/compare.xhtml

```diff
@@ -76,28 +79,32 @@
                 <h:outputText value="File elements are identical" style="font-weight:bold; padding: 5px 10px; " rendered="#{uploadExcelDiffController.richDiffTree.status=='EQUALS'}"/>
             </div>
 
-            <table style="width: 100%">
-                <tr>
-                    <td>#{msg['ws.compare.file1']}</td>
-                    <td>#{msg['ws.compare.file2']}</td>
-                </tr>
-                <tr>
-                    <td valign="top">
-                        <br />
-                        <a4j:outputPanel ajaxRendered="true">
-                            <rules:tableEditor id="tableEditor1" table="#{uploadExcelDiffController.table1}"
-                                editable="false" filters="#{uploadExcelDiffController.filter1}" />
-                        </a4j:outputPanel>
-                    </td>
-                    <td valign="top">
-                        <br />
-                        <a4j:outputPanel ajaxRendered="true">
-                            <rules:tableEditor id="tableEditor2" table="#{uploadExcelDiffController.table2}"
-                                editable="false" filters="#{uploadExcelDiffController.filter2}" />
-                        </a4j:outputPanel>
-                    </td>
-                </tr>
-            </table>
+            <h:panelGroup id="diffCompare">
+                <table style="width: 100%">

```

Comment:
Please replace table elements with div. Cause tables look ugly and it's challenging to define strict size to them because they always shrink and unclenches by default